src/ansible_navigator/_version.pyi

```diff
@@ -0,0 +1,16 @@
+# fmt: off
+# isort: skip_file
+"""Stub (interface) file for ``_version.py``.
+
+Provides type hint PEP 484 information for the ``_version.py``
+file because it is generated by setuptools_scm and may
+not exist when type checking is don and will not
+container type hint information.
+"""
+
+from typing import Union
+
+from .configuration_subsystem import Constants
```

Comment:
This seems inaccurate. I only see `Constants` in `cli.py` but not `_version.py`

Reply:
yeah, mypy picks up it being assigned a Constant in cli.py

```
(venv) x1 ➜  ansible-navigator git:(scm_version) mypy
src/ansible_navigator/cli.py:36:19: error: Incompatible types in assignment (expression has type "Constants", variable has type "str")  [assignment]
        __version__ = Constants.NOT_SET
                      ^
```

Reply:
no longer relevant :)